qutebrowser/config/configdata.yml

```diff
@@ -746,6 +746,17 @@ content.xss_auditing:
     Suspicious scripts will be blocked and reported in the inspector's
     JavaScript console.
 
+content.mute:
+  default: false
+  type: Bool
+  supports_pattern: true
+  desc: >-
+    Mute new web pages.
+
+    It's off by default meaning that new web pages might play sounds.
+    It's possible to set this configuration differently for different
+    URLs.
```

Comment:
I don't think this paragraph is necessary:

- "It's off by default meaning that new web pages might play sounds." seems clear - the default value is in the docs, and that it's allowed to play sounds because it's not muted should be obvious.
- "It's possible to set this configuration differently for different URLs." is already added to the docs based on the `supports_pattern: true`